spec/requests/api/v1/auth/passwords/update_spec.rb

```diff
@@ -0,0 +1,32 @@
+describe 'PUT api/v1/auth/passwords', type: :request do
+  let(:user) { create(:user, must_change_password: true, password: 'password') }
+  let(:password) { 'abcd1234' }
+  let(:params) { { password: password } }
+  let(:headers) { }
+
+  subject do
+    put api_v1_user_password_path, params: params, headers: headers, as: :json
+    response
```

Comment:
the subject should not end with `response`. If you want to use if for the one line `is expected to` i'd rather we did it like 
```
       specify do
         request

         expect(response).to have_http_status(:success)
      end
```